.ci/check_unsupported.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Non-zero exit code is what we want to check
+set +e
+
+# Enable capturing the non-zero exit status of coverage instead of tee
+set -o pipefail
+
+sed -i.bak -E '/^(mypy-lang|guess-language-spirit)/d' bear-requirements.txt
```

Comment:
can you add comment on why this line was added

Reply:
sorry I forgot to copy that comment from `.travis.yml`
now that this unsupported code oath is not using pip to install dependencies, `guess-language-spirit` also doesnt install , as setuptools can only find the latter version of the package, and then it complains that the version isnt the same as we require.